### PR TITLE
Uyuni 2024.02

### PR DESCRIPTION
**Pull request by raulillo82**
PR to update the site with the release of Uyuni 2024.02.

### CODE REVIEW

**Review by deneb-alpha — COMMENTED** [review]
I would put  more emphases on the CVEs

File: pages/news.html [hunk 1]
```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+	  <p><strong>16/02/2024</strong> - Uyuni 2024.02 is now available with server image for aarch64, mgradm start/stop/restart commands and CVE fixes. Check the <a href='../pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
Comment (raulillo82) [inline pages/news.html:91]:
```suggestion
	  <p><strong>16/02/2024</strong> - Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='../pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is now also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```

File: index.html [hunk 2]
```diff
@@ -231,23 +231,25 @@ <h3>Latest news</h3>
...
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">
-              <span>31/01/2024</span>
+              <span>16/02/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.01 is now available with openEuler 22.03 support, reboot required indicator for non-SUSE distributions... For uyuni-tools in deb-based systems, please check the release announcement. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.02 brings important CVEs fixes, please, read carefully the <a href='pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is available also for aarch64 (ARM64), and mgradm provides new start/stop/restart commands</p>
```
Comment (raulillo82) [inline index.html:241]:
```suggestion
              <p>Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```

**Review by deneb-alpha — APPROVED** [review]
LGTM

File: index.html [hunk 3]
```diff
@@ -231,23 +231,25 @@ <h3>Latest news</h3>
...
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">
-              <span>31/01/2024</span>
+              <span>16/02/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.01 is now available with openEuler 22.03 support, reboot required indicator for non-SUSE distributions... For uyuni-tools in deb-based systems, please check the release announcement. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.02 is now available with server image for aarch64, mgradm start/stop/restart commands and CVE fixes. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```
[the remark this answers is not in the document]
Comment (raulillo82) [reply index.html]: I committed the change and then slightly tuned it (punctuation, mainly). Then I applied the same change in the news page. Please double check it still looks good.

File: index.html [hunk 4]
```diff
@@ -231,23 +231,25 @@ <h3>Latest news</h3>
...
           <!-- News box Start -->
-          <div class="news-box">
+	  <div class="news-box">
             <div class="news-box-title">
-              <span>31/01/2024</span>
+              <span>16/02/2024</span>
             </div>
             <div class="news-box-content">
-              <p>Uyuni 2024.01 is now available with openEuler 22.03 support, reboot required indicator for non-SUSE distributions... For uyuni-tools in deb-based systems, please check the release announcement. Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+              <p>Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```
Comment (juliogonzalez) [inline index.html:241]: Question of tests, but I'd mention the reboot flag instead of start/stop/restart for mgradm :-)

File: pages/news.html [hunk 5]
```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+	  <p><strong>16/02/2024</strong> - Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='../pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is now also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```
Comment (juliogonzalez) [inline pages/news.html:91]: Same as before, but no need to accept my idea ;-)

**Review by juliogonzalez — COMMENTED** [review]
LGTM. Left just two comments in case you want to adjust the highlights (not mandatory)

File: index.html (hunk 4 above)
Comment (raulillo82) [inline index.html:241]:
```suggestion
              <p>Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is now also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```

Comment (raulillo82) [reply index.html]: That was already included in 2024.01

File: pages/news.html (hunk 5 above)
Comment (raulillo82) [reply pages/news.html:91]: Already included (and announced) in 2024.01.

**Review by juliogonzalez — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (juliogonzalez) [reply index.html]: Ah, right. I probably got confused by SUSE Manager release notes.

**Review by deneb-alpha — APPROVED** [review]
LGTM

**Pull request merged**